packages/cli/src/index.ts

```diff
@@ -427,6 +427,11 @@ program
         'Specifies the first day of the week (used by week-related date functions). 0 (Monday) to 6 (Sunday)',
         parseStartOfWeekArgument,
     )
+    .option(
+        '--skip-dbt-compile',
+        'Skip `dbt compile` and deploy from the existing ./target/manifest.json',
+        false,
+    )
```

Comment:
The option and code changes look good 👍 
I think we should add this option to all commands that compile the project.  `preview`, `start-preview` and `validate`